app/presenters/character_presenter.rb

```diff
@@ -23,18 +23,18 @@ def as_json(options={})
   end
 
   def multi_gallery_json
-    galleries = character.galleries.ordered.includes(:icons)
+    galleries = character.galleries.ordered.includes(:ordered_icons)
     galleries_json = galleries.map do |gallery|
       {
         name: gallery.name,
-        icons: gallery.icons.order('LOWER(keyword)')
+        icons: gallery.ordered_icons
```

Comment:
This was loading icons twice because it was ordering them – include the ordered relation and use that instead